content/neuvector-prime/azure/contents.lr

```diff
@@ -239,19 +226,51 @@ Logs from the containers of pods of the job
 kubectl logs -l job-name=<job name> -n <namespaces>
 ```
 
-Troubleshooting Neuvector Usage Record: 
 
+###Neuvector Usage Record Not found: 
 
+Error:
 Error from server (NotFound): cspadapterusagerecords.susecloud.net "neuvector-usage" not found
  Check Configuration, Retrieve generated configuration csp-config
     
 
+Solution:
 ```
 kubectl get cm -n neuvector csp-config -oyaml
 ```
 
+
 if a configuration is not listed, you can dig into the root cause by checking the pod status and log (Refer Jobs and Pods section).
 
+
+###Multiple extensions of same type: 
+
+Error:
+
+ERROR TYPE
+Multiple extensions of same type is not allowed at this scope. (Code: ValidationFailed)
+
+Solution:
+
+AKS cluster already have extension with the same type. Uninstall the extension and re-deploy with the same cluster.
+
+   !["Uninstall extension"](./uninstall.png)
+
+###Resource already existing in your cluster: 
+
+Error:
+
+Helm installation failed : Resource already existing in your cluster : Recommendation Manually delete the resource(s) that currently exist in your cluster and try installation again. To delete these resources run the following commands: kubectl delete <resource type> -n <resource namespace> <resource name> : InnerError [rendered manifests contain a resource that already exists. Unable to continue with install: ServiceAccount "csp" in namespace "neuvector" exists and cannot be imported into the current release: invalid ownership metadata; annotation validation error: key "meta.helm.sh/release-name" must equal "test-nv2-reinstall": current value is "testnv2-plan"]
```

Comment:
Is this the full error message or is this both the message and the recommendation? I'm a bit confused what comes from the Azure error and what is a recommended response since there is a separate solution in the next block

Reply:
that's full error message which gives solution. 

Also created solution section, where you can follow instruction as suggested by the Error kubectl command or use azure console uninstall option (screenshot)

Reply:
hmm maybe quote the error?